src/dpsvalidationservice/dfcmsords.yaml

```diff
@@ -0,0 +1,82 @@
+swagger: '2.0'
+info:
+  title: DFCRMS
+  version: 1.0.0
+host: localhost
+basePath: /ords/deva/dfcmsords/web
+schemes:
+- http
+produces:
+- application/json
+securityDefinitions:
+  basicAuth:
+    type: basic
+    description: HTTP Basic Authentication.
+tags:
+- name: DFCRMS
+paths:
+  /caseSequenceNumber:
+    get:
+      tags:
+      - DFCRMS
+      security:
+      - basicAuth: []
+      description: Get case sequence number
+      responses:
+        '200':
+          description: output of the endpoint
+          schema:
+            $ref: '#/definitions/caseSequenceNumberResponse'
+        '400':
+          description: output of the endpoint
+          schema:
+            $ref: '#/definitions/errorMessage'
+      parameters:
+      - name: driver_license_no
+        in: query
+        type: string
+        description: Implicit parameter
+      - name: surname_code
+        in: query
+        type: string
+        description: Implicit parameter
+  /health:
```

Comment:
what happens if health returns a non 200?

Reply:
nothing right now, we don't use this endpoint